### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by zmLGBBM** [AtomLinter/linter-clang#64]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:247:11)
Package.activateNow (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:218:14)
<unknown> (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:203:30)
Package.measure (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:147:15)
```

**Comment by k2b6s9j** [comment]
I already fixed this. Waiting on @Snake231088 to push the update to APM.

**Comment by k2b6s9j** [comment]
Also a dupe of #60

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-clang/pull/61 and published in version `2.25.2`

**Comment by zmLGBBM** [comment]
@k2b6s9j @kevinsawicki Thanks for the update ;-)
